llvm/test/tools/llvm-objcopy/ELF/gap-fill-order.test

```diff
@@ -1,16 +1,19 @@
-# RUN: yaml2obj %s >%t
+# RUN: yaml2obj %s -o %t
```

Comment:
I should probably have mentioned this earlier: in llvm-objcopy, where we have edge case behaviours to test for a given feature, we usually fold them into a single test file (i.e. I'd add the body of this test to `gap-fill.test`).

If it's necessary to have different YAML docs, simply use the `--docnum` option to specify which one to use.